types/ckeditor__ckeditor5-core/src/plugincollection.d.ts

```diff
@@ -7,6 +7,13 @@ import ContextPlugin from "./contextplugin";
 import Editor from "./editor/editor";
 import Plugin, { PluginInterface, LoadedPlugins } from "./plugin";
 
+declare global {
+    namespace CKEditor {
+        // tslint:disable-next-line:no-empty-interface
+        interface Plugins {}
+    }
+}
+
```

Comment:
For the record, this is the most "controversial" line. I'm fine with it, but it would be great if some DT mantainer can approve it too.